src/one_dragon_qt/view/setting/setting_env_interface.py

```
@@ -68,26 +67,19 @@ def _init_code_group(self) -> SettingCardGroup:
             icon=FluentIcon.APPLICATION, title='代码源', content='国内无法访问GitHub则选择Gitee',
             options_enum=RepositoryTypeEnum
         )
-        self.repository_type_opt.value_changed.connect(lambda: self.ctx.git_service.update_git_remote())
+        self.repository_type_opt.value_changed.connect(lambda: self.ctx.git_service.update_remote())
         code_group.addSettingCard(self.repository_type_opt)
 
-        self.git_method_opt = ComboBoxSettingCard(
-            icon=FluentIcon.SYNC, title='拉取方式', content='不懂什么是ssh就选https',
-            options_enum=GitMethodEnum
+        self.auto_update_opt = SwitchSettingCard(
+            icon=FluentIcon.SYNC, title='自动更新', content='使用exe启动时，自动检测并更新代码',
```

Ruff (0.14.4)

String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)